src/components/TableBodyContainer.js

```diff
@@ -5,31 +5,33 @@ import compose from 'recompose/compose';
 import mapProps from 'recompose/mapProps';
 import getContext from 'recompose/getContext';
 
-import { visibleRowIdsSelector, classNamesForComponentSelector, stylesForComponentSelector } from '../selectors/dataSelectors';
+//import { visibleRowIdsSelector, classNamesForComponentSelector, stylesForComponentSelector } from '../selectors/dataSelectors';
 
 const ComposedTableBodyContainer = OriginalComponent => compose(
   getContext({
     components: PropTypes.object,
     selectors: PropTypes.object,
   }),
-  connect((state, props) => ({
-    visibleRowIds: visibleRowIdsSelector(state),
-    className: classNamesForComponentSelector(state, 'TableBody'),
-    style: stylesForComponentSelector(state, 'TableBody'),
-  })),
+  connect(
+    (state, props) => ({
+      visibleRowIds: props.selectors.visibleRowIdsSelector(state),
+      className: props.selectors.classNamesForComponentSelector(state, 'TableBody'),
+      style: props.selectors.stylesForComponentSelector(state, 'TableBody'),
+    })
+  ),
   mapProps(props => {
-    const { components, ...otherProps } = props;
+    const { components, selectors, ...otherProps } = props;
     return {
       Row:  props.components.Row,
       ...otherProps,
     };
   }),
 )(({Row, visibleRowIds, style, className}) => (
   <OriginalComponent
-    rowIds={visibleRowIds}
-    Row={Row}
-    style={style}
-    className={className}
+  rowIds={visibleRowIds}
+  Row={Row}
+  style={style}
+  className={className}
```

Comment:
This component can pass-thru `props` directly, like the other components:

```javascript
)(props => <OriginalComponent {...props} />);
```

Reply:
Clarification: please rename `visibleRowIds` to `rowIds` so `props` can pass-thru directly.